week-3/A-array-find/exercise.js

```diff
@@ -1,13 +1,16 @@
-/* 
-  You are given an array of test scores.
-  Any result over 60 is a pass. 
-  Using .find() (and any other array methods necessary), find the lowest pass score
+/*
+ You are given an array of names.
+ Using .find(), we'd like to find the first name which starts with A and is longer than 7 letters.
 */
 
-var testScores = [90, 50, 100, 25, 81, 66, 80];
-var lowestPassScore = findLowestPassScore(testScores);
+// write your code here
 
-console.log(lowestPassScore);
+var names = ["Rakesh", "Alexandra", "Annam", "Mikey", "Karim", "Ahmed"];
+var longNameThatStartsWithA = names.find(findLongNameThatStartsWithA);
+function findLongNameThatStartsWithA(name) {
+  return name[0].includes("A") && name.length > 7;
```

Comment:
This is very correct, but you can make a small readability improvement. When using `name[0]` that will return a string only 1 length long. The `.includes()` method will search the **whole** string to check if any of the string includes the "A" letter.

So that means that using `.includes()` is a bit unnecessary to check a 1 length string. It might be better to directly compare `name[0]` against `"A"`.

Reply:
There is another small improvement that you could make. What happens if you change the `names` array so that "Alexandra" starts with a lower case "a" (`"alexandra"`)? How could we fix this?

Reply:
the code can be changed like this for both things from
return name[0].includes("A") && name.length > 7;
to 
return name[0]==="A" && name.length > 7;